app/models/EpicTests.scala

```diff
@@ -39,6 +39,18 @@ case class TickerSettings(
   copy: TickerCopy
 )
 
+sealed trait SeparateArticleCountType extends EnumEntry
```

Comment:
This pattern works and is how we've been doing enums elsewhere, so no need to change it now - but I just discovered there is now a better way:
https://stackoverflow.com/a/66967469